website/addons/s3/static/s3NodeConfig.js

```diff
@@ -317,6 +317,9 @@ ViewModel.prototype.openCreateBucket = function() {
                                 '<label class="col-md-4 control-label" for="bucketName">Bucket Name</label> ' +
                                 '<div class="col-md-8"> ' +
                                     '<input id="bucketName" name="bucketName" type="text" placeholder="Enter bucket name" class="form-control" autofocus> ' +
+                                    '<div>' +
+                                        '<span id="errorMessage" ></span>' +
```

Comment:
`errorMessage` is too generic; it's too likely that the same ID might be used elsewhere on the page. Can you make it more specific, like `bucketModalErrorMessage`?